Black-Market/Black-Market/Scenes/SignIn/SignInView.swift

```diff
@@ -0,0 +1,117 @@
+                    signUpCardView
+                }
+                .padding(UIConstants.Defaults.padding)
+                .navigationTitle("")
+                .navigationDestination(for: Dest.self) {
+                    switch $0 {
+                    case .home:
+                        Text("Home Destination Here", comment: "Home screen title")
+                    case .signUp:
+                        SignUpView()
+                    }
+                }
+            }
+        }
+    }
+    
+    private var mainCardView: some View {
+        VStack(spacing: UIConstants.Defaults.spacing) {
+            Image.blackmarketLogo
+            TextField("Type your email", text: $email)
+                .textFieldStyle(
+                    MarketTexfield(
+                        style: .constant(.default),
+                        title: String(localized: "Email", comment: "Email title")
+                    )
+                )
+                .onChange(of: email) { newValue in
+                    isEmailValid = newValue.isValidEmail
+                    style = (isEmailValid && isPasswordValid) ? .filled : .disabled
+                }
+
+            SecureField("Type your password", text: self.$password)
+                .textFieldStyle(
+                    MarketTexfield(
+                        style: .constant(.password),
+                        title: String(localized: "Password", comment: "Password title"),
+                        placeholderText: "Type your password",
+                        fieldValue: self.$password
+                    )
+                )
+                .onChange(of: password) { newValue in
+                    isPasswordValid = newValue.isValidPassword
+                    style = (isEmailValid && isPasswordValid) ? .filled : .disabled
+                }
+            
+            MarketButton(style: $style, action: {
+                path.append(.home)
+            }, title: String(localized: "Log In", comment: "Log In button title"))
+            
+            Text(tappableText)
+                .handleTappableLinks()
+        }
+        .padding(UIConstants.Defaults.padding)
+        .background(
+            .white,
+            in: RoundedRectangle(cornerRadius: UIConstants.Defaults.cornerRadius, style: .continuous)
+        )
+    }
+    
+    private var signUpCardView: some View {
+        VStack(spacing: UIConstants.Defaults.spacing) {
+            Text("Don’t have an account?").font(.custom("Open Sans", size: 14))
+            MarketButton(style: .constant(.plain), action: {
+                path.append(.signUp)
+            }, title: String(localized: "Sign Up", comment: "Sign Up button title"))
+        }
+        .padding(UIConstants.Defaults.padding)
+        .background(.white,
+                    in: RoundedRectangle(cornerRadius: UIConstants.Defaults.cornerRadius,
+                                         style: .continuous))
```

Comment:
pls, use the localization. Also consider the comments from the previous PR about the font, colors and constant centralization

Reply:
I'm already using catalog string localization.

<img width="1168" alt="Screenshot 2023-07-30 at 8 07 50 PM" src="https://github.com/JUNIOR140889/Black-Market/assets/42948405/a3c11f0f-ffe7-4b44-8935-f179d62b9d50">

Also, I added files for fonts and colors.